docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,131 @@
+# Developers Clear Containers 3.0 Install
+
+This is not a end-user guide, this is for people that are eager to try Clear Containers 3.0 and want to build it from source and are familiar doing so. Basic programming knowlege  and be confortable to work with the Linux command line is expected.
```

Comment:
This text changes the scope of the document and clarifies its audience. Thank you @gorozco1 ! Please keep the line length in mind and may I suggest changing the text to:
This guide is not intended for end-users. Instead, this guide provides instructions for any developers eager to try Clear Containers 3.0 and who want to build Clear Containers from the source code and are familiar with the process. Basic programming knowledge and familiarity with the Linux command line are expected.